pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -955,6 +955,10 @@
     dependencies = with self; [ nui-nvim ];
   };
 
+  none-ls-nvim = super.none-ls-nvim.overrideAttrs {
+    dependencies = with self; [ plenary-nvim ];
```

Comment:
when there is only one dependency, I prefer `self.plenary-nvim` which is shorter.

Reply:
Done !